pkg/vadmin/create_db_vc.go

```diff
@@ -75,7 +75,7 @@ func (v *VClusterOps) genCreateDBOptions(s *createdb.Parms, certs *HTTPSCerts) v
 	}
 
 	// Additional configuration parameters for create db.
-	opts.ConfigurationParameters = s.ConfigurationParams
```

Comment:
I'm trying to understand the vclusterops changes again. We still have this parameter (`ConfigurationParameters`). Why would you want to use `ConfigurationParameters` vs `CommunalStorageParameters`? They both seem to do the same thing. In fact, we combine the maps when we call the bootstrap catalog op. We have masking for sensitive parms, but it doesn't seem like it matters where it comes from.

Reply:
I agree that they both seem to do the same thing. AT is using both ConfigurationParameters and CommunalStorageParameters, and our vcluter mimic AT, there is some reasons e.g. For usability -- users understand better on the effect of the parameters they pass in

Reply:
Thanks Chinh